cmake/compiler_settings.cmake

```diff
@@ -84,11 +85,33 @@ elseif (MSVC)
   if (MSVC_VERSION VERSION_LESS 1910)
     message(FATAL_ERROR "Only Visual Studio 2017 and above supported!")
   endif ()
+
+  # Explicitly set C/CXX flags rather than using the defaults. This uses the defaults
```

Comment:
We should link to the defaults.

Reply:
The comment explains which defaults I left off. I'll add why, but it is very much intentional.

Reply:
Added comment with justification and added back RTC1. All errors that triggered RTC failures have been fixed.

Reply:
I think we should link to wherever you found the defaults (I usually consult the CMake source code for the version we're tied to). But it's not an easy thing for most people to find.

Reply:
I just printed the values. I’ll look for a source file and link it

Reply:
Added more details and source file for reference.

Reply:
Thanks!